src/Serializer/DoctrineDenormalizer.php

```diff
@@ -40,12 +49,20 @@ public function denormalize($data, $class, $format = null, ?array $context = nul
             return $data;
         }
 
+        // Find lookup key for given class
+        $findKey = $this->getClassLookupKey($class);
+
+        // If a lookup key exists, then find by lookup key
+        if ($findKey !== null && isset($data[$findKey]) === true) {
+            return $this->findOneBy($class, [$findKey => $data[$findKey]]);
```

Comment:
I wonder if it'd be more flexible if $findKey was a tuple of (entityIdKey, requestIdKey) ?

Right now if we want it to be different (like externalId) this process wouldnt work?

Reply:
Denormalizer now supports multiple key mappings, and incorrect configuration will trigger exception